sdk-wasm/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "solana-sdk-wasm"
+description = "Solana SDK Wasm"
+documentation = "https://docs.rs/solana-sdk"
```

nit: since this is really specific to using `wasm-pack` to generate an npm package, let's call out the js part:

```suggestion
name = "solana-sdk-wasm-js"
description = "Solana SDK Wasm JS"
documentation = "https://docs.rs/solana-sdk-wasm-js"
```

---

Should I rename the folder to `sdk-wasm-js` as well?

---

Ok, I've renamed the folder too.